### DISCUSSION TITLE
PDF output?

### DISCUSSION THREAD

**Discussion by RohanAlexander** [rstudio/pointblank#424] [General]
Hey Rich,

Love the package. Just wondering if there's any plans to potentially add PDF output options at some point?

Rohan

**Comment by rich-iannone** [comment]
Thanks, Rohan! I actually hadn’t considered LaTeX output for any of the reporting. It would be a huge undertaking if I were to do it, and I have to wonder how many would take advantage of this functionality. Which reporting piece would be most useful in LaTeX form for you?

**Reply by RohanAlexander** [reply]
Thank you for the consideration Rich. 

No need to do it just for me, especially if it's a lot of work. I've got a work-around for my PDFs just by taking a screenshot and including a figure, and that works well. :)
